tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -343,12 +343,7 @@ system_health/test_system_health.py::test_service_checker_with_process_exit:
 #######################################
 vxlan/test_vxlan_ecmp.py:
   skip:
-    reason: "VxLAN ECMP test is not yet supported on multi-ASIC platform"
+    reason: "VxLAN ECMP test is not yet supported on multi-ASIC platform. Also this test can only run on 4600c and 8102."
     conditions:
-      - "is_multi_asic==True"
+      - "(is_multi_asic==True) or (platform not in ['x86_64-mlnx_msn4600c-r0', 'x86_64-8102_64h_o-r0'])"
```

Comment:
There are bunch of other Nvidia devices that support this feature.

Reply:
@roysr-nv : I setup this check based on the previous check, which only had one testcase. I have kept the platforms listed the same as before. If other platforms are to be included, pls raise a seperate PR.